kubejs/server_scripts/unify.js

```diff
@@ -256,10 +256,11 @@ events.listen('recipes', function(e) {
 
     //Craft only metals
     //Multiple Mods crafted only (no ore)
-    unifyCraftMetal('bronze', 'mekanism:ingot_bronze', 'mekanism:dust_bronze', 'mekanism:block_bronze', 'mekanism:nugget_bronze');
+    //unifyCraftMetal('bronze', 'mekanism:ingot_bronze', 'mekanism:dust_bronze', 'mekanism:block_bronze', 'mekanism:nugget_bronze'); //Replaced/Returned to unifying on thermal's bronze
```

Comment:
Any particular reason not to just remove this line? Git won't forget it, if you ever need it back.

Reply:
Git will remember it better if it has the comment in there, so that somebody knows *why* it was removed.